core/kazoo_call/src/kapps_call.erl

```diff
@@ -136,9 +136,9 @@
 -export([default_helper_function/2]).
 
 -export([start_recording/1, start_recording/2
-        ,mask_recording/1
-        ,unmask_recording/1
-        ,stop_recording/1
```

Comment:
add arity 2 for `mask_recording` , `unmask_recording` instead of deleting the existing as you're proposing for stop_recording